<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use ECSPrefix20220117\Vendor\Package\SomeNamespace\{
    ECSPrefix20220117\SubnamespaceOne\ClassA,
    ECSPrefix20220117\SubnamespaceOne\ClassB,
    ECSPrefix20220117\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use ECSPrefix20220117\Vendor\Package\SomeNamespace\{
    <em>ECSPrefix20220117\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    ECSPrefix20220117\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
